app/src/renderer/components/governance/LiProposal.vue

```diff
@@ -0,0 +1,139 @@
+<template lang="pug">
+li.li-proposal
```

Comment:
this shouldn't be an `li` anymore - `table-validators` is a proper html table - should be the same for proposals

Reply:
oh ok I'll update it

Reply:
if you want, you can open an issue to refactor `table-validator` to be more generalized for proposals as well and leave this how it is for now.